src/app/components/reminder.js

```diff
@@ -3,36 +3,47 @@ import notify from './../base/notify';
 import { notes } from './../constants/notes';
 import { messages } from './../constants/messages';
 
-export default () => {
-    const APIURl = 'http://localhost:5003/reminders';
-    const day = document.querySelector('[data-reminder="day"]');
-    const time = document.querySelector('[data-reminder="time"]');
-    const setBtn = document.querySelector('[data-reminder="set"]');
-    setBtn.disabled = true;
-    let SELECTEDDAY = 'Select day';
-    let SELECTEDTIME = 'Select time';
-
-    day.addEventListener('change', (event) => {
+export default class Reminder {
+    constructor(container) {
+        this.container = container;
+        this.apiUrl = this.container.getAttribute('data-apiUrl');
+        this.day = this.container.querySelector('[data-reminder="day"]');
+        this.time = this.container.querySelector('[data-reminder="time"]');
+        this.setBtn = this.container.querySelector('[data-reminder="set"]');
```

Comment:
this is not particularly wrong, but it's better to select elements based on `class` than `data-attribute`. [Data attributes are for storing some custom data](https://intuio.at/blog/dont-use-data-attributes-to-find-html-elements-with-js/)